src/listeners/reactionPin.ts

```diff
@@ -0,0 +1,41 @@
+import {Events, Listener, PieceContext} from '@sapphire/framework';
+import {getSettings} from '../database/models/SettingsModel';
+import colors from '../utils/colorUtil';
+import {BediEmbed} from '../lib/BediEmbed';
+import {MessageReaction, User} from 'discord.js';
+import logger from '../utils/loggerUtil';
+
+module.exports = class PinReactionListener extends Listener {
+  constructor(context: PieceContext) {
+    super(context, {
+      event: Events.MessageReactionAdd,
+    });
+  }
+
+  public async run(messageReaction: MessageReaction, user: User) {
+    const {message} = messageReaction;
+    const {guild, guildId} = message;
+
+    if (!guild || messageReaction.emoji.name != '📌') return;
```

Comment:
I feel like this should maybe be custom? Or at the very least a const at the top of the code lol